### DISCUSSION TITLE
"Access Violation" error

### DISCUSSION THREAD

**Discussion by nvskumar-m** [AntonelloDN/Morpho#24] [Bugs]
Hello,

I have installed the ENVI-Met and Morpho updates recently, and I am encountering an error as provided in the snapshot below.

I tried to compose the geometry with respect to the grid as shown below, yet I am receiving the error once the model is opened in Spaces application, especially when I want to look at a 3D model.

Kindly guide me to address this concern.

<img width="1700" alt="Screenshot 2024-01-09 094653" src="https://github.com/AntonelloDN/Morpho/assets/54622340/599b4eed-17a9-416c-bed5-0f0daeee53c6">
<img width="559" alt="Screenshot 2024-01-09 094442" src="https://github.com/AntonelloDN/Morpho/assets/54622340/c9cc847a-8afb-46a2-9126-1b265f243dea">
<img width="785" alt="Screenshot 2024-01-09 094538" src="https://github.com/AntonelloDN/Morpho/assets/54622340/cbe87b5d-0906-4b2a-894d-cac67e774199">
<img width="1366" alt="Screenshot 2024-01-09 094620" src="https://github.com/AntonelloDN/Morpho/assets/54622340/993bba4f-6373-4563-8819-0b3e72e48f30">

**Comment by AntonelloDN** [comment]
Hi @nvskumar-m, Unfortunately the error message generated by Spaces is not user-friendly. A tip can be adding incrementally objects to find out which entity generates the issue. E.g. buildings first; buildings + soils and so on. 
Usually it is related to an error in the INX file.
